iotdb-client/cli/src/assembly/resources/tools/load-tsfile.bat

```diff
@@ -96,14 +98,44 @@ EXIT /B %ret_code%
 
 @REM -----------------------------------------------------------------------------
 :finally
-
-set PARAMETERS=%h_parameter% %p_parameter% %u_parameter% %pw_parameter% %load_dir_parameter% %sg_level_parameter% %verify_parameter% %on_success_parameter%"
-echo %PARAMETERS%
-
 echo start loading TsFiles, please wait...
-"%JAVA_HOME%\bin\java" %JAVA_OPTS% -cp %CLASSPATH% %MAIN_CLASS% %PARAMETERS%
-set ret_code=%ERRORLEVEL%
 
-ENDLOCAL
+if exist "%load_dir_parameter%" (
+    if not exist "%load_dir_parameter%\" (
+        call :load %load_dir_parameter%
+    ) else (
+        call :recursiveFunction %load_dir_parameter%
+    )
+) else (
+    echo %load_dir_parameter% is not valid.
+    exit /b 1
+)
 
+set ret_code=0
+echo end loading TsFiles
+ENDLOCAL
 EXIT /B %ret_code%
+
+:recursiveFunction
+for /r "%1" %%F in (*.tsfile) do (
```

Comment:
Try all the files instead of only "*.tsfile" to be consistent with the core implementation.